addons/web_editor/static/src/components/upload_progress_toast/upload_service.js

```diff
@@ -98,8 +98,8 @@ export const uploadService = {
                 // limited by bandwidth.
                 for (const sortedFile of sortedFiles) {
                     const file = progressToast.files[sortedFile.progressToastId];
-                    const dataURL = await getDataURLFromFile(sortedFile);
                     try {
+                        const dataURL = await getDataURLFromFile(sortedFile);
```

Comment:
What will be the error shown to the user in this case? If the error shown is not helpful, can't  we do in the `base` module diff of this PR reject a proper error message like "Empty file" instead of adding it to the existing condition return empty Reject?